src/metabase/util/ssh.clj

```diff
@@ -118,6 +122,7 @@
   "Close a running tunnel session"
   [details]
   (when (and (use-ssh-tunnel? details) (ssh-tunnel-open? details))
+    (log/tracef "Closing SSH tunnel: %s" (:tunnel-session details))
```

Comment:
Added some extra logging here to make debugging this a little easier in the future